agent/api/task/task_windows.go

```diff
@@ -214,6 +215,11 @@ func (task *Task) addFSxWindowsFileServerResource(
 	return nil
 }
 
+// For Windows task run via the docker DaemonManager, we will run them as ContainerAdministrator
+func (task *Task) getTaskUser(container *container.Container) string {
+	return "ContainerAdministrator"
```

Comment:
Convert this into a constant

Reply:
It is already present in https://github.com/aws/amazon-ecs-agent/blob/27ca59a0a052c2e8a6e330899f02969ba15dbe66/agent/ecscni/namespace_helper_windows.go#L36. Maybe export that one.